tests/featuredContent.test.ts

```diff
@@ -35,7 +35,7 @@ describe('Featured content resolver', () => {
   beforeEach(async () => {
     mongoose = await connectMongoose(process.env.MONGO_URL!)
     aggregates = await buildAggregates()
-    server = await createServer(mongoose, aggregates)
+    server = await createServer(mongoose, aggregates, 'https://sumer-dev-2.joystream.app/query/server/graphql')
```

Comment:
I'm guessing that I'm not supposed to pass this URL to tests, but I didn't know how to do it properly. Feel free to add any suggestion

Reply:
Hmm maybe we require this to come from the env as well? Not sure yet